aws_batch/pipeline.py

```diff
@@ -506,17 +564,31 @@ def filter(self, record):
         self.last = record.relativeCreated
         return True
 
+def percent_str(percent):
+    try:
+        return "%3.1f" % percent
+    except:
+        return str(percent)
+
+def logger_echo(level, msg)
+    print level + ": " + msg
+
+def logger_info(msg)
+    logger_echo("INFO", msg)
+    logger_info(msg)
```

Comment:
did you mean `logger.info(msg)` inside `logger_info`? (I think we should still produce the logfile in S3 as well)

Reply:
yes that is the result of a careless find-and-replace!   thank you for catching it

Reply:
as it is, it would have crashed on bottomless recursion right away :P)